src/kv/store.h

```diff
@@ -702,6 +702,8 @@ namespace kv
 
           LOG_DEBUG_FMT(
             "Batching {} ({})", last_replicated + offset, data_shared->size());
+
+          // TODO: Encryptor
```

Comment:
That's where we want to notify the encryptor of new secrets, right?

Reply:
Absolutely. I clearly wrote it there as a note to my future self and forgot to remove it.